.drone.yml

```
@@ -107,7 +107,7 @@ steps:
   - test_sbt
   - test_java11
   commands:
-  - ./project/scripts/sbtPublish ";project dotty-bootstrapped ;publishSigned ;sonatypeBundleRelease ;project tasty-core-scala2; publishSigned ;sonatypeBundleRelease"
+  - ./project/scripts/sbtPublish ";project dotty-bootstrapped ;publishSigned ;sonatypeBundleRelease"
```

If the problem is that some artifacts get uploaded twice, we can clean the locally published artifacts in the middle:
```suggestion
  - ./project/scripts/sbtPublish ";project dotty-bootstrapped ;publishSigned ;sonatypeBundleRelease ;sonatypeBundleClean ;project tasty-core-scala2; publishSigned ;sonatypeBundleRelease"
```

---

I don't think that'll help. The problem is not the staging area but maven central. By the time `sonatypeBundleRelease` finished, the staging area is promoted to maven central, and we can't clean up that.

---

If you figure something out, please let me know because I think I have been having the same problem. https://github.com/ekrich/sconfig/issues/61